### PR TITLE
Change Relationship to Basic Patient Information

### PR DESCRIPTION
**Pull request by dsamojlenko**
# What this does

- Creates the Basic Patient Information screen (previously Relationship)

Note: currently directs to "Expedited" section, will direct back to Dashboard when that's available.

## Known issue
Our nunjucks templates don't support passing in an i18n string with mixin, so for now just passing in the translated string (ie, passing in `__('something')` instead of `something`) ... this has the side effect of dynamically-created i18n strings being added to the locale files. 

Please just ignore these entries for now. We will come back to this later.

### CODE REVIEW

**Comment by jeffmaher** [comment]
@sidewalkballet : Can you do the design review on this?

File: api/controllers/RelationshipController.js [hunk 1]
```diff
@@ -29,10 +29,12 @@ module.exports = {
...
+      req.session.medicalReport.relationshipStarted = req.body.relationshipStarted;
+      req.session.medicalReport.firstTreatmentDate = req.body.firstTreatmentDate;
       req.session.medicalReport.visitNumber = req.body.visitNumber;
       req.session.medicalReport.lastVisitDate = req.body.lastVisitDate;
-      req.session.medicalReport.firstTreatmentDate = req.body.firstTreatmentDate;
+      req.session.medicalReport.stopWorking = req.body.stopWorking;
+      req.session.medicalReport.stopWorkingWhen = req.body.stopWorkingWhen;
       dataStore.storeMedicalReport(req.session.medicalReport);
 
       res.redirect(sails.route('expedited'));
```
Comment (CalvinRodo) [inline api/controllers/RelationshipController.js]: Should redirect to the Dashboard

**Comment by CalvinRodo** [comment]
I was able to address all issues except for the extra space, that'll take a bit more digging I think.

**Comment by CalvinRodo** [comment]
Fixed the issue with the conditional logic.

**Pull request merged**